docs/_functions/domain/CAA.md

```diff
@@ -0,0 +1,34 @@
+---
+name: CAA
+parameters:
+  - name
+  - tag
+  - value
+  - modifiers...
+---
+
+CAA adds a CAA record to a domain. The name should be the relative label for the record. Use `@` for the domain apex.
+
+Tag can be one of "issue", "issuewild" or "iodef". Value depends on the tag, and is automatically quoted.
```

Comment:
What I meant is that the DNS record requires a quote around the value (eg. `issue "letsencrypt.org"`) which is done automatically (i.e. `CAA("@", "issue", "letsencrypt.org")` rather than `CAA("@", "issue", "\"letsencrypt.org\"")`).

English is not my primary language though, so please feel free to suggest improvements. Or, I think this can be omitted since the auto quoting behavior is similar to TXT records. (i.e. leave the DNS technical details to the library)